integration/airflow/tests/integration/airflow/logs/latest

```diff
@@ -1 +1 @@
-/opt/airflow/logs/2021-09-03
\ No newline at end of file
+/opt/airflow/logs/2021-09-14
```

Comment:
Should `/logs/latest` file be checked in?

Reply:
Actually, yes - we need file to indicate that logs folder has necessary permissions. The file is actually used there, so nicely fits that purpose.